snuba/util.py

```diff
@@ -636,15 +637,20 @@ def wrapper(*args, **kwargs):
     return decorator
 
 
-def decode_part_str(part_str):
+class Part(NamedTuple):
+    date: datetime
```

Comment:
This should probably be a `date` and not a `datetime`, but I haven't checked to see if that would break anything that expects it to be one and not the other.